detection.py

```diff
@@ -128,3 +130,41 @@ def detect_drift(
                     print(
                         f"Drift detected in feature {feature} before and after {time_cutoffs[ix]}"
                     )
+
+    def _remove_missing_values(
```

Comment:
This method not just removes the missing values, also checks the change in the proportion. Please, create separate methods for each objective. (FYI, the tests ignore the missing values, so I'm not sure there is a need for removing them)